src/search.sjs

```diff
@@ -64,20 +64,42 @@ if (input.features === false) {
 
 var industries = {};
 var features = {};
+var found = [];
 
-if (input.industries === true) {
+if (input.firstLoad === true) {
   industries = jsearch.facets([jsearch.facet('Industries', 'industry').slice(0,100)]).result();
+  features = jsearch.facets([jsearch.facet('Features', 'features').slice(0,215)]).result();
 }
 
-if (input.features === true) {
-  features = jsearch.facets([jsearch.facet('Features', 'features').slice(0,215)]).result();
+if (input.industries !== '') {
+  // function to find all users in a given industry
+  // 'ind' as a string parameter represents the industry
+  // outputs array of GeoJSON objects
+  xdmp.log("finding");  
```

Comment:
Some log statements are worth leaving in the code, but if you're going to, it should have more information. This looks like a debugging log statement, so it should probably be removed.